docs/build-instructions.md

```diff
@@ -65,6 +65,22 @@ When installing, you must minimally select all Visual C++ components.
 
 #### Building with terminal
 
+Since Ping-Viewer v2.2.0, the project now uses CMake to simplify the build process.
+For more information about CMake with Qt based projects, check the [Qt CMake manual](https://doc.qt.io/qt-5/cmake-manual.html);
+
+1. Use CMake to configure the project
+    - `cmake -B build -DCMAKE_BUILD_TYPE=Debug`
```

Comment:
Add a note to ensure Qt5 is on `PATH`?

Reply:
Also make sure Qt5.15 is installed (current instructions are for 5.12). 

For Mac can use `brew install qt@5`, then `export PATH="/usr/local/opt/qt@5/bin:$PATH"` before doing `cmake -B build...`. Alternatively can permanently add to path by doing `echo 'export PATH="/usr/local/opt/qt@5/bin:$PATH"' >> ~/.zshrc`, then restarting terminal before building.

Reply:
Can you check the new instructions ? There a new tip about different installation environment. 
I also fixed a problem with the CMakeFile.txt for the project, so you should fetch the latest master branch now.
The problem appears to be with the Qt documentation where QtQuickCompiler configuration was necessary to use in a different way, the problem related to this module should not happen and the current PR should solve any problem related to the Qt5 search process.